adminguide.rst

```diff
@@ -320,6 +322,52 @@ job queue and record of past jobs.
     If this happens it is best to stop Flux, remove the content.sqlite file,
     and restart.
 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+flux-accounting configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In order to support fairshare and multi-factor priority ordering of user jobs, the flux-accounting package must be present and configured for the system instance. This allows for the management of users and banks on a cluster, job priorities to be calculated using multiple factors and uses job record information from users to calculate and maintain fairshare values.
```

Comment:
Suggestion: now that flux-accounting was introduced above, perhaps just replace this with
> If `flux-accounting` was installed, its database should be initialized, and the job manager configured to automatically load the multifactor priority plugin provided by `flux-accounting`.

Reply:
Okay! I like this suggestion, thank you - I can make this change.

> the job manager configured to automatically load the multifactor priority plugin provided by `flux-accounting`.

I didn't know this was possible! Does the job manager automatically load the multifactor priority plugin provided by `flux-accounting` via TOML file?

Reply:
Yes that is the purpose of the TOML config.